Assets/HoloToolkit/Utilities/Scripts/Tagalong.cs

```diff
@@ -291,7 +291,7 @@ private bool AdjustTagalongDistance(Vector3 cameraPosition, out Vector3 newPosit
                             {
                                 Light clonedLight = Instantiate(DebugPointLight, closestHit.point, Quaternion.identity) as Light;
                                 clonedLight.color = Color.red;
-                                DestroyObject(clonedLight, 1.0f);
+                                Destroy(clonedLight, 1.0f);
```

Comment:
Do we want to check if application is editor and use destroy immediate if true?

Reply:
Possibly but this PR was only to future proof the current functionality rather than change it.
Better we have a separate PR to review "Destroy"s use through out the project?